### PR TITLE
add workdir on step to allow relative path into test files

### PR DESCRIPTION
**Pull request by marcaudefroy**
Currently, when venom runs tests placed into subdirectory, the relative path into files test isn't well used. Indeed, executors aren't access to workdir about a test file.  This PR fix this issue.

### CODE REVIEW

**Comment by marcaudefroy** [comment]
Waiting before merge...

File: executors/dbfixtures/executor.go [hunk 1]
```diff
@@ -62,9 +64,10 @@ func (e Executor) Run(testCaseContext venom.TestCaseContext, l venom.Logger, ste
 	// Load and import the schemas in the database
 	// if the argument is specified.
 	if len(e.Schemas) != 0 {
+
```
Comment (wI2L) [inline executors/dbfixtures/executor.go]: ??

File: executors/dbfixtures/executor.go [hunk 2]
```diff
@@ -42,7 +43,8 @@ type Result struct {
 }
 
 // Run implements the venom.Executor interface for Executor.
-func (e Executor) Run(testCaseContext venom.TestCaseContext, l venom.Logger, step venom.TestStep) (venom.ExecutorResult, error) {
+func (e Executor) Run(testCaseContext venom.TestCaseContext, l venom.Logger, step venom.TestStep, workdir string) (venom.ExecutorResult, error) {
+
```
Comment (wI2L) [inline executors/dbfixtures/executor.go]: ??

Comment (marcaudefroy) [reply executors/dbfixtures/executor.go]: PR is in progress

**Comment by yesnault** [comment]
This will break existing users who use relative import in their yml files

Proposal: add optional version in yml files.

if version:2 -> use workdir, if no version (=version:1 -> use behaviour as today)

This will avoid to break test on existing users, but let you to add this cool feature in yml version 2.

**Comment by marcaudefroy** [comment]
Yvonnick : version added.
To test it, there are two file into dbfixture. A file on version 1 (no version on testsuite) and a file on version 2.

File: process_files.go [hunk 3]
```diff
@@ -107,6 +107,18 @@ func (v *Venom) readFiles(filesPath []string) (err error) {
 
 		ts.Name += " [" + f + "]"
 
+		if ts.Version == "2" {
```
Comment (sguiheux) [inline process_files.go]: if not startwith 1 will be better and will avoid to modify this line for the next version

**Pull request merged**